classes/class-wc-connect-nux.php

```diff
@@ -212,6 +212,7 @@ public function set_up_nux_notices() {
 						array( $this, 'ajax_get_jetpack_connect_url' )
 					);
 					add_action( 'admin_notices', array( $this, 'show_banner_before_connection' ) );
+					add_action( 'admin_notices', array( $this, 'remove_jetpack_notices' ), 1 );
```

Comment:
Should this get called inside `show_banner_before_connection()` instead? There is still a check inside `show_banner_before_connection()` that determines if the NUX notice should display.